components/o3-button/stories/sustainable-views/button.stories.tsx

```diff
@@ -17,8 +17,11 @@ export default {
 	],
 	argTypes: Object.assign(ButtonStories.Button.argTypes, {
 		theme: {
-			options: ['inverse', 'mono'],
-			control: { type: 'radio' },
+			options: ['default', 'inverse'],
+			mapping: {
+				default: '',
```

Comment:
To align with the terminology we are using in Figma?
```suggestion
			options: ['standard', 'inverse'],
			mapping: {
				standard: '',
```